### PR TITLE
[ADD] stock_reserve_no: no automatic stock reservation

### PR DESCRIPTION
**Pull request by dreispt**

### CODE REVIEW

File: stock_reserve_no/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    "summary": "Do not auto-reserve stock for Inventory Transfers",
+    "version": "14.0.1.0.0",
+    "license": "AGPL-3",
+    "website": "https://github.com/OCA/stock-logistics-warehouse",
+    "author": "Open Source Integrators, Odoo Community Association (OCA)",
+    "category": "Inventory/Inventory",
+    "depends": ["stock"],
+    "data": ["views/stock_picking_views.xml"],
+    "installable": True,
+    "development_status": "Beta",
```
Comment (rousseldenis) [inline stock_reserve_no/__manifest__.py:15]: That's the default

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Hi @dreispt. Thanks for this.

I would have changed the name by 'stock_no_automatic_reservation' or something else more explicit

**Comment by rousseldenis** [comment]
@dreispt

**Comment by dreispt** [comment]
I don't think I ended up using this module, so I'm closing the PR.
If someone else find this feature interesting, you're welcome to pick it up.
Thanks!

**Pull request closed without merging**